cdr-aggregation/notebooks/README.md

```diff
@@ -0,0 +1,7 @@
+# Notebook organization
+
+The [aggregation_master.py](https://github.com/worldbank/covid-mobile-data/tree/cdr-master/cdr-aggregation/notebooks/aggregation_master.py) script is currently set to run all flowminder, priority and scaled indicators. Additional custom indicators are left out.
+
+The [aggregation_master.ipynb](https://github.com/worldbank/covid-mobile-data/tree/cdr-master/cdr-aggregation/notebooks/aggregation_master.py) notebook does the same and can be used for data exploration, too.
+
+The [aggregation_master.ipynb](https://github.com/worldbank/covid-mobile-data/tree/cdr-master/cdr-aggregation/notebooks/aggregation_master.py) notebook is customised for databricks.
```

Comment:
Why is this file linked to a second time. Can we just move it up to the previous line like this?
```suggestion
The [aggregation_master.ipynb](https://github.com/worldbank/covid-mobile-data/tree/cdr-master/cdr-aggregation/notebooks/aggregation_master.py) notebook does the same and can be used for data exploration, too. This notebook is customised for databricks.
```

Reply:
Just fixed that - the wrong file was referenced. Thanks!